cmd/sonobuoy/app/run.go

```diff
@@ -93,7 +106,7 @@ func submitSonobuoyRun(cmd *cobra.Command, args []string) {
 		plugins[i] = plugin.Name
 	}
 
-	if len(plugins) > 0 {
+	if len(plugins) > 0 && runflags.wait == 0 {
```

Comment:
What was this added for? Seems unrelated to this spinner change, maybe just a leftover of tinkering?

Reply:
Done.